### DISCUSSION TITLE
Time with clock emoji

### DISCUSSION THREAD

**Discussion by MitPitt** [SilentVoid13/Templater#304] [Templates Showcase]
Current time with the closest possible corresponding clock emoji (they have one for each half-hour).

I have next to none javascript knowledge, so the code is probably not optimal.

```
<%*
	const string = ["🕛","🕧","🕐","🕜","🕑","🕝","🕒","🕞","🕓","🕟","🕔","🕠","🕕","🕡","🕖","🕢","🕗","🕣","🕘","🕤","🕙","🕥","🕚","🕦"];
	var currentdate = new Date();
	var time = Number((2 * (currentdate.getHours() + currentdate.getMinutes() / 60)).toFixed()) % 24;
	tR += tp.date.now("LT ") + string[time];
%>
```

Will produce something like:
`9:00 🕘`
`17:12 🕔`
`23:14 🕚`
`23:15 🕦`
`23:46 🕛`

**Comment by wxnnvs** [comment]
Noice, exactly what i was looking for

**Comment by Yu1e** [comment]
Thank you, excellent idea and implementation!
